src/oneTrust/getAssessment.ts

```diff
@@ -0,0 +1,24 @@
+import { Got } from 'got';
+import { OneTrustGetAssessmentResponse } from './types';
+
+/**
+ * Retrieve details about a particular assessment.
+ *
+ * @param param - the information about the OneTrust client and assessment to retrieve
+ * @returns details about the assessment
+ */
+export const getAssessment = async ({
+  oneTrust,
+  assessmentId,
+}: {
+  /** The OneTrust client instance */
+  oneTrust: Got;
+  /** The ID of the assessment to retrieve */
+  assessmentId: string;
+}): Promise<OneTrustGetAssessmentResponse> => {
+  const { body } = await oneTrust.get(
+    `api/assessment/v2/assessments/${assessmentId}/export?ExcludeSkippedQuestions=false`,
+  );
+
+  return JSON.parse(body) as OneTrustGetAssessmentResponse;
```

Comment:
nit: should use an io-ts codec to validate here

Reply:
Created the codecs here https://github.com/transcend-io/privacy-types/pull/205/files and battle-tested them agains the OneTrust API responses. Caught lots of stuff! Thanks for the suggestion.